src/io/flutter/settings/FlutterSettings.java

```diff
@@ -269,11 +270,16 @@ public boolean isEnableHotUiInCodeEditor() {
   }
 
   public boolean isEnableEmbeddedBrowsers() {
-    return getPropertiesComponent().getBoolean(enableEmbeddedBrowsersKey, true);
+    // TODO(helinx): Change default value once JxBrowser is compatible with MacOS 11.
+    return getPropertiesComponent().getBoolean(enableEmbeddedBrowsersKey, !SystemInfo.isMacOSBigSur);
```

Comment:
I think the 2nd param to this method is the default value if none is set. If they've previously set a value here, they'll get that value, which I assume we won't want for big sur.

I'd suggest just changing the start of this method to something like `if (SystemInfo.isMacOSBigSur) return false;`